src/stream/src/executor/aggregation/mod.rs

```diff
@@ -342,6 +342,59 @@ pub fn generate_agg_schema(
     Schema { fields }
 }
 
+/// Infer column desc for state table.
+/// The column desc layout is
+/// [ `group_key` (only for hash agg) / `sort_key` (only for simple agg) / `value`(the agg call
+/// return type)].
+/// This is the Row layout insert into state table.
+/// For different agg call, different executor (hash agg or simple agg), the layout will be
+/// different.
+pub fn generate_column_descs(
```

Comment:
Just out of curiosity, will this be generated by meta service or frontend eventually?

Reply:
I think so, serveral days ago discussed with @st1page .